study-builder/studies/osteo/patches/dob-validations.conf

```diff
@@ -0,0 +1,149 @@
+{
+  "items": [
+    {
+      "activityCode": "CONSENT",
+      "validations": [
+        {
+          "messageTemplate": {
+            "templateType": "HTML",
+            "templateText": "$osteo_consent_dob_validation",
+            "variables": [
+              {
+                "name": "osteo_consent_dob_validation",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": "(TBD) You're not old enough for self consent!"
```

Comment:
Still waiting on actual message to use, and how many validations to add, but the code is ready for review.